### ISSUE TITLE
VTKCellViewer fails with petsc solvers

### ISSUE THREAD

**Issue by pavaninguva** [usnistgov/fipy#702]
Thank you for the great work implementing PETSc! I tried to run my code using the PETSc library and while the same code works for PySparse/ Trilinos and I can use the VTKCellViewer to export the simulation data as VTK files, when I use PETSc, I end up with the following problem: 
```
Traceback (most recent call last):
  File "finitevolume.py", line 102, in <module>
    vw = VTKCellViewer(vars=(x_a, mu_AB))
  File "/home/pavaninguva/miniconda3/envs/fipy2/lib/python3.8/site-packages/fipy/viewers/vtkViewer/vtkViewer.py", line 49, in __init__
    self.dataset = self._makeDataSet(mesh)
  File "/home/pavaninguva/miniconda3/envs/fipy2/lib/python3.8/site-packages/fipy/viewers/vtkViewer/vtkCellViewer.py", line 16, in _makeDataSet
    return mesh.VTKCellDataSet
  File "/home/pavaninguva/miniconda3/envs/fipy2/lib/python3.8/site-packages/fipy/meshes/abstractMesh.py", line 1013, in VTKCellDataSet
    cvi = self._orderedCellVertexIDs.swapaxes(0, 1)
  File "/home/pavaninguva/miniconda3/envs/fipy2/lib/python3.8/site-packages/fipy/meshes/abstractMesh.py", line 833, in _orderedCellVertexIDs
    return self._calcOrderedCellVertexIDs()
  File "/home/pavaninguva/miniconda3/envs/fipy2/lib/python3.8/site-packages/fipy/meshes/uniformGrid2D.py", line 561, in _calcOrderedCellVertexIDs
    return ids.reshape((4, self.numberOfCells), order="FORTRAN")
ValueError: Non-string object detected for the array ordering. Please pass in 'C', 'F', 'A', or 'K' instead
```
I am not too sure what this means. Thanks for the help!

**Comment by guyer** [comment]
What version of numpy do you have installed?

I doubt this has anything to do with PETSc per se. Rather, this is an error from numpy that used to be a deprecation warning. I was going to deal with it once the PETSc release was out, but it looks like I'll need to do an immediate bug fix release.

**Comment by pavaninguva** [comment]
Hi! Sorry for the delay! My numpy is 1.18.1 from pypi_0.

**Comment by guyer** [comment]
OK, that's what I'd guess. numpy 1.18 is still pretty new and not yet available from conda-forge, which is the only installation pathway we test (we call it "recommended" for a reason).

**Comment by guyer** [comment]
That's not to say this won't get fixed (#703), just that it's not surprising to me that we haven't seen this breakage.
